libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/drillDown.spec.ts

```diff
@@ -136,4 +159,88 @@ describe("Drilling", { tags: ["post-merge_integrated_bear"] }, () => {
             drillModal.getTable().getColumnValues(0).should("deep.equal", ["2011"]);
         });
     });
+
+    describe("Basic drill down",{ tags: ["checklist_integrated_bear"] }, () => {
+        const tableData = [
+            {
+                title: "table has row column measure",
+                values: ["CompuSci", "$4,163,871.80", "$11,418,823.89"]
+            },
+            {
+                title: "table only row and column",
+                values: ["CompuSci"]
+            },
+            {
+                title: "table only rows and measures",
+                values: ["CompuSci", "$15,582,695.69"]
+            },
+            {
+                title: "table only rows",
+                values: ["CompuSci"]
+            }
+        ]
+    
+        it("Should drill down on table with one drillable",  () => {
+            Navigation.visit("dashboard/dashboard-table-drill-down");
+            tableData.forEach((ins, index) => {
+                new Widget(index).waitTableLoaded().getTable().click(0, 0);
+                drillModal
+                    .getTitleElement()
+                    .should("have.text", ins.title + " › Direct Sales");
+    
+                ins.values.forEach((assertValue, index) => {
+                    drillModal.getTable().hasCellValue(0, index, assertValue)
+                })
+                drillModal.close();
+            });
+        });
+    
+        it("Should drill down on charts that have two drillable attributes", () => {
+            const firstDrillValue = ["617.00", "1,903.00", "3,526.00", "2,046.00", "41.00", "220.10", "386.70", "281.00"];
```

Comment:
please move all definition like `const firstDrillValue = ["617.00", "1,903.00", "3,526.00", "2,046.00", "41.00", "220.10", "386.70", "281.00"];` to the top.